src/parser/jobs/blm/modules/Procs.js

```diff
@@ -146,7 +142,7 @@ export default class Procs extends Module {
 		}
 
 		tracker.current.stop = endTime
-		if (tracker.current.stop >= tracker.current.wear && countDrops) {
+		if (tracker.current.stop - tracker.current.start >= getStatus(statusId).duration * 1000 && countDrops) {
```

Comment:
I would probably keep the durations array at the top of the file and do the math there.